torch/autograd/grad_mode.py

```diff
@@ -386,7 +387,10 @@ class _unsafe_preserve_version_counter(_DecoratorContextManager):
 
     """
 
-    def __init__(self, tensors: Union[torch.Tensor, Tuple[torch.Tensor, ...]]) -> None:
+    def __init__(
+        self,
+        tensors: Union[torch.Tensor, typing.Tuple[torch.Tensor, ...]],  # noqa: UP006
```

Comment:
Why not use `tuple` here?

Reply:
Oh - that one looks like I probably could have changed it. I'll try it.